src/util/image/MapCreator.java

```diff
@@ -66,6 +69,13 @@
   }
 
   public static void main(final String[] args) {
+      try {
+        UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());//Fallback Look and Feel
+      } catch (ClassNotFoundException | InstantiationException | IllegalAccessException
+          | UnsupportedLookAndFeelException e) {
+        ClientLogger.logError("Default Look and Feel could not be applied", e);
```

Comment:
This is a warning, not an error. Also no need for a full stack trace in this instance. Can do:
` ClientLogger.logError("Default Look and Feel could not be applied: "+ e.getMessage());`